lib/internal/priority_queue.js

```diff
@@ -11,7 +11,7 @@ const {
 // just a single criteria.
 
 module.exports = class PriorityQueue {
-  #compare = (a, b) => a - b;
+  #compare = (a, b) => (a > b ? 1 : a < b ? -1 : 0);
```

Comment:
I know such a common static code can commonly be compact as an idiom. How about, other members?